app/controllers/content_controller.rb

```diff
@@ -21,8 +19,7 @@ def show
       end
     end
 
-    doc = Govspeak::Document.new(@page.markdown, sanitize: true, allowed_elements: ContentController::ALLOWED_TAGS)
-    @markdown = doc.to_html
+    @markdown = MarkdownToHtml.new.translate_markdown(@page.markdown)
```

Comment:
Govspeak is not markdown, it doesnt work the same so we need to avoid using the term as its misleading, so no new classes or variables should be added which use the term markdown, can you rename it to Govspeak please